static/templates/ssids.html

```diff
@@ -4,29 +4,103 @@
 	<meta charset="utf-8" />
 	<meta name="viewport" content="width=device-width, initial-scale=1" />
 
-	<title>SSIDs</title>
-
-	<link rel="stylesheet" href="/static/css/application.css" />
+	<title>Select WIFI</title>
 
+    <link rel="stylesheet" href="/static/css/application.css" />
 </head>
+
 <body>
-    <form>
-    <table>
-        <thead>
-            <tr>
-              <th style="text-align:center" colspan="2">Select Wifi to connect to:</th>
-            </tr>
-        </thead>
-        <tbody>
-        {{with .Ssids}}
-            {{range .}}
-            <tr>
-                <td style="text-align:right"><input type="radio" name="essid" value="{{.Ssid}}"/></td><td style="text-align:left">{{.Ssid}}</td>
-            <tr>
-            {{end}}
-        {{end}}
-        </tbody>
-    </table>
-    </form>	
+    <div class="wrapper">
+        <div id="main-content" class="inner-wrapper">
+            <div class="row no-border" id="grid">
+                <h2>Select WIFI to connect to</h2>
+                <fieldset>
+                <div class="twelve-col">
+                <table>
+                <tbody>
+                {{range $i, $network := .Networks}}
+                <tr>
+                    <th scope="row">
+                        <label class="collapse" for="radio{{$i}}">{{$network.Ssid}}</label>
+                        <input id="radio{{$i}}" type="radio" name="c1">
+                        <div class="six-col">
+                        <fieldset>
+                        <ul class="no-bullets">
+                            <li>
+                                <input type="hidden" id="ssid{{$i}}" value="{{$network.Ssid}}"/>
+                                <input type="password" id="password{{$i}}"/>
+                            </li>
+                            <li>
+                                <input type="checkbox" id="showpassword{{$i}}" onchange="show_password({{$i}})"/>
+                                <label for="showpassword">show password</label>
+                            </li>
+                            <li>
+                                 <div id="alert{{$i}}" class="cheshire box" style="background-color: #eee">
+                                    <h3>Caution!</h3>
+                                     <p>While connecting to an external network you won't be able to manage device. Proceed?.</p>    
```

Comment:
I prefer a something a little more direct for now: "Continuing will disconnect you from the device by taking down its wifi network. This page will be unavailable. After continuing, you may connect to the device through its new wifi connection."